### PR TITLE
Created Entity classes or LoginModel

### PR DESCRIPTION
**Pull request by sachreetkaur**
Added Pspcl.Core project and Domain folder that contain all entity classes and also created loginModel as well.

### CODE REVIEW

File: Pspcl/Pspcl.Core/Class1.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+﻿namespace Pspcl.Core
+{
+    public class Class1
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Core/Class1.cs]: Do we need this ?

File: Pspcl/Pspcl.Models/LoginModel.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Pspcl.Models
+{
+    public class LoginModel
+    {
+        [Key]
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Models/LoginModel.cs]: This is a model, not an entity. Wo dont specify relationships and constrainsts in models

File: Pspcl/Pspcl.Models/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Models/Properties/launchSettings.json]: In a class library project, we dont add launch settings to open it in browser.

File: Pspcl/Pspcl.Services/Properties/launchSettings.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Services/Properties/launchSettings.json]: There should not be a launch settings for a library project

File: Pspcl/Pspcl.Models/LoginModel.cs (hunk 2 above)
Comment (sachreetkaur) [reply Pspcl/Pspcl.Models/LoginModel.cs]: This is the separate LoginModel that I have to create in Pspcl.Models.

Comment (sandeepkumarlng) [reply Pspcl/Pspcl.Models/LoginModel.cs]: You can create models, but my question is about constraints you are putting in a model

Comment (sachreetkaur) [reply Pspcl/Pspcl.Models/LoginModel.cs]: Ok I will remove these.

File: Pspcl/Pspcl.Models/LoginModel.cs [hunk 5]
```diff
@@ -11,8 +11,6 @@ namespace Pspcl.Models
 {
     public class LoginModel
     {
-        [Key]
-        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
         public int Id { get; set; }
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Models/LoginModel.cs]: What is the purpose of Id column here ?

File: Pspcl/Pspcl.Core/Domain/UserRolesMapping.cs [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+using System.ComponentModel.DataAnnotations.Schema;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Data;
+
+namespace Pspcl.Core.Domain
+{
+    public class UserRolesMapping
```
Comment (sandeepkumarlng) [inline Pspcl/Pspcl.Core/Domain/UserRolesMapping.cs:12]: We dont need Id column here. Also, add Role entity like you have added User

**Review by sandeepkumarlng — APPROVED** [review]

**Pull request merged**